app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -395,6 +396,19 @@ def save_container_volumes_inventory(container_group, hashes, target = nil)
     store_ids_for_new_records(container_group.container_volumes, hashes, :name)
   end
 
+  def save_additional_attributes_inventory(entity, hashes, target = nil)
```

Comment:
I'd like to see this merge with the [save_custom_attributes_inventory](https://github.com/ManageIQ/manageiq/blob/9db73c860bf9a5cf6ad030db881177ed3a47035d/app/models/ems_refresh/save_inventory.rb#L285) method since `additional_attributes` is really just an [alias for `custom_attributes`](https://github.com/ManageIQ/manageiq/blob/9db73c860bf9a5cf6ad030db881177ed3a47035d/app/models/ems_refresh/save_inventory.rb#L285).  But, this has two major differences:

1.  `additional_attributes` vs. `custom_attributes` in the call to `save_inventory_multi`
2. this version calls `store_ids_for_new_records` whereas `save_custom_attributes_inventory` does not ... and I'm not sure why.